tests/test_create_dataset_manifest.py

```diff
@@ -300,6 +300,48 @@ def test_object_detection_bbox_format_LTRB(self):
             self.assertEqual(dataset_manifest.images[0].labels, [[0, 10, 10, 100, 100]])
             self.assertEqual(dataset_manifest.images[1].labels, [[0, 100, 100, 200, 200], [1, 20, 20, 200, 200]])
 
+    def test_image_caption(self):
+        manifest_dict = {
+            "images": [{"id": 1, "file_name": "train_images.zip@honda.jpg"},
+                       {"id": 2, "file_name": "train_images.zip@kitchen.jpg"}],
+            "annotations": [
+                {"id": 1, "image_id": 1, "caption": "A black Honda motorcycle parked in front of a garage."},
+                {"id": 2, "image_id": 1, "caption": "A Honda motorcycle parked in a grass driveway."},
+                {"id": 3, "image_id": 1, "caption": "A black Honda motorcycle with a dark burgundy seat."},
+                {"id": 4, "image_id": 1, "caption": "Ma motorcycle parked on the gravel in front of a garage."},
+                {"id": 5, "image_id": 1, "caption": "A motorcycle with its brake extended standing outside."},
+                {"id": 6, "image_id": 2, "caption": "A picture of a modern looking kitchen area.\n"},
+                {"id": 7, "image_id": 2, "caption": "A narrow kitchen ending with a chrome refrigerator."},
+                {"id": 8, "image_id": 2, "caption": "A narrow kitchen is decorated in shades of white, gray, and black."},
+                {"id": 9, "image_id": 2, "caption": "a room that has a stove and a icebox in it"},
+                {"id": 10, "image_id": 2, "caption": "A long empty, minimal modern skylit home kitchen."}
+            ],
+            "type": "captions"
```

Comment:
image_caption or captions? Is this test working? Thanks

Reply:
The key "type" is not used in the dataloader, so it can be removed. I have removed it